.github/workflows/ci-schedule-compatibility.yaml

```
@@ -0,0 +1,66 @@
+name: CI Schedule Compatibility Testing Workflow
```

The workflow name seems not straightforward enough. 

```suggestion
name: APIServer compatibility
```